src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="Apple">
```

Comment:
```suggestion
      <a class="header__logo" href="#Apple">
```
try to add `#` before every href, or replace it with # or /. 
now when we press the link the error page is loading, try to fix it